lib/metrics_dashboard/health.rb

```diff
@@ -0,0 +1,75 @@
+class MetricsDashboard
```

Comment:
I think classes within classes aren't good practice if they are referred to outside the parent class.  I think a better way would be to have a module called metrics, and everything inside that module, leaving MetricsDashboard (or Metrics::MetricsDashboard) as just a class doing what it does.